mmcv/runner/hooks/momentum_updater.py

```diff
@@ -49,28 +52,33 @@ def _set_momentum(self, runner, momentum_groups):
                 elif 'betas' in param_group.keys():
                     param_group['betas'] = (mom, param_group['betas'][1])
 
-    def get_momentum(self, runner, base_momentum):
+    def get_momentum(self, runner, base_momentum) -> float:
```

Comment:
type hint for `base_momentum`

Reply:
I have added this before, but is declined by mypy because the function signature in [`OneCycleMomentumUpdaterHook.get_momentum`](https://github.com/triwahyuu/mmcv/blob/54292936e3c189fc21d908c6855cdbd49e998e1b/mmcv/runner/hooks/momentum_updater.py#L563) is different compared to other child class from this.  
Any idea how to resolve this better?

Reply:
allright, If the signature of the function has changed, we don't need to add Type Hints.

Reply:
okay, I think I can resolve this then